EULA.md

```diff
@@ -0,0 +1,288 @@
+*The Mass Open Cloud Alliance (MOC Alliance) has developed a production
+public cloud to provide domain researchers with predictable low-cost
+resources and facilitator support while enabling academic researchers
+and developers in the open source community to participate in close
+interactions between research, development, and production operations.
+Housed at the Boston University Hariri Institute for Computing and
+Computational Science & Engineering, the MOC Alliance is a unique
+collaborative effort between higher education, non-profit entities and
+industry (MOC Alliance Collaborators). For a full list of the MOC
+Alliance Collaborators, please visit*
+*[here](https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/)*.
+
+**MASS OPEN CLOUD ALLIANCE (MOC ALLIANCE) END USER LICENSE AGREEMENT**
+
+This End User License Agreement (the “Agreement”) applies to all uses of
+software and services rendered by and in relation to the Mass Open Cloud
+Alliance (the “Services”). Each Service is provided and managed by the
+MOC Alliance Collaborator(s) identified [here](#service_name) (each a
+“Provider”).
+
```

Comment:
Suggestion from OGC to change line 15-19: 
This End User License Agreement (the “Agreement”) applies to all uses of software and services rendered by and in relation to the Mass Open Cloud Alliance (the “Services”). Each Service is provided and managed by the MOC Alliance Collaborator(s) identified here (each a “Provider”). The Providers may change this Agreement at any time, and we will try to let you know as soon as possible about any new changes. If you do not wish to accept the new Agreement, you must immediately stop using the Services and terminate this Agreement pursuant to Section 12.

Reply:
to adress items below